spec/models/partners/item_request_spec.rb

```diff
@@ -26,6 +27,20 @@
     it { should validate_numericality_of(:quantity).only_integer.is_greater_than_or_equal_to(1) }
     it { should validate_presence_of(:name) }
     it { should validate_presence_of(:partner_key) }
+
+    it "should only be able to use organization's request units" do
```

Comment:
So this would change to "should only be able to use item's request units"  with the corresponding changes to the test...